airflow/models.py

```diff
@@ -764,6 +770,8 @@ def command(
         cmd += "--local " if local else ""
         cmd += "--pool {pool} " if pool else ""
         cmd += "--raw " if raw else ""
+        cmd += "--airflow_cfg \"{pickled_cfg}\" " if airflow_cfg else ""
```

Comment:
Are we passing the configuration as an argument so that that we don't have to make `airflow.cfg` visible to the running user?